<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmSystemSetUpClass</class>
 <widget class="QWidget" name="frmSystemSetUpClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>292</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>QMacVisual</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/QMacVisual/Icon/other_478.ico</normaloff>:/QMacVisual/Icon/other_478.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #bcc2bc;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dfd5cf, stop: 0.15 #acbbac, stop: 0.85 #acbbac,stop:1 #dfd5cf);
border:1px solid rgb(150, 150, 170);</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="documentMode">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>基本设置</string>
        </attribute>
        <widget class="QCheckBox" name="checkProgramSelfStart">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>40</y>
           <width>123</width>
           <height>23</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
    color: rgb(30, 30, 30);
	border-style: flat;
	background: transparent;
}
QCheckBox::indicator {
    width: 17px;
    height: 17px;    
}
QCheckBox::indicator::unchecked {
    image: url(:/res/ico/untick.png);
}
QCheckBox::indicator::checked {
    image: url(:/res/ico/tick.png);
};</string>
         </property>
         <property name="text">
          <string>开机自启动</string>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkProcessSelfStart">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>40</y>
           <width>229</width>
           <height>23</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
    color: rgb(30, 30, 30);
	border-style: flat;
	background: transparent;
}
QCheckBox::indicator {
    width: 17px;
    height: 17px;    
}
QCheckBox::indicator::unchecked {
    image: url(:/res/ico/untick.png);
}
QCheckBox::indicator::checked {
    image: url(:/res/ico/tick.png);
};</string>
         </property>
         <property name="text">
          <string>启动后流程自动开始执行</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkSignOutTime">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>122</y>
           <width>230</width>
           <height>23</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
    color: rgb(30, 30, 30);
	border-style: flat;
	background: transparent;
}
QCheckBox::indicator {
    width: 17px;
    height: 17px;    
}
QCheckBox::indicator::unchecked {
    image: url(:/res/ico/untick.png);
}
QCheckBox::indicator::checked {
    image: url(:/res/ico/tick.png);
};</string>
         </property>
         <property name="text">
          <string>程序无操作自动登出时间：</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinSignOutTime">
         <property name="geometry">
          <rect>
           <x>286</x>
           <y>121</y>
           <width>56</width>
           <height>26</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget{color: rgb(85, 0, 255);background-color: rgb(180, 180, 210);}
QSpinBox::up-button{
    border:transparent;
}
QSpinBox::down-button{
    border:transparent;
}
QSpinBox::up-arrow{
    image:url(:/res/ico/up.png);
}
QSpinBox::down-arrow{
    image:url(:/res/ico/down.png);
}</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>60</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>346</x>
           <y>122</y>
           <width>41</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;
background-color: rgba(255, 255, 255, 0);</string>
         </property>
         <property name="text">
          <string>分钟</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>日志设置</string>
        </attribute>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>40</y>
           <width>123</width>
           <height>19</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;
background-color: rgba(255, 255, 255, 0);</string>
         </property>
         <property name="text">
          <string>日志保存天数：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>40</y>
           <width>23</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;
background-color: rgba(255, 255, 255, 0);</string>
         </property>
         <property name="text">
          <string>天</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinLogTime">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>37</y>
           <width>56</width>
           <height>26</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget{color: rgb(85, 0, 255);background-color: rgb(180, 180, 210);}
QSpinBox::up-button{
    border:transparent;
}
QSpinBox::down-button{
    border:transparent;
}
QSpinBox::up-arrow{
    image:url(:/res/ico/up.png);
}
QSpinBox::down-arrow{
    image:url(:/res/ico/down.png);
}</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>365</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>45</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
